config/prod/ahayden-ubuntu-test.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: ahayden-ubuntu-test
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["22"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "hackathon_testing"
+  # The resource owner
+  OwnerEmail: "aaron.hayden@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "computevpc"
+  KeyName: "scicomp"
+  # AMIId: "ami-0ca7208f904dc1874"      # AWS linux 2 AMI 2.0.20200108 x86_64 HVM gp2
+  #AMIId: "ami-0d576d466a205f6db"      # Ubuntu, 18.04 LTS, amd64 bionic image build on 2019-02-12
+  AMIId: "ami-0d03e44a2333dea65"      # Canonical Ubuntu image, 18.04 LTS, amd64 bionic image https://aws.amazon.com/marketplace/server/configuration?productId=3b73ef49-208f-47e1-8a6e-4ae768d8a333
```

Comment:
that's AMI for ubuntu 16.04 not 18.04.  The 18.04 quick start one is `ami-07ebfd5b3428b6f4d`

Reply:
Are you sure? The marketplace shows it as 18.04 2019113 to me